core-java-modules/core-java-collections-4/pom.xml

```diff
@@ -20,6 +20,11 @@
             <artifactId>commons-lang</artifactId>
             <version>${commons-lang.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.12.0</version>
```

Comment:
Let's use a property (commons-lang3.version) for this version number

Reply:
Fixed.